backend/config/plugins.js

```diff
@@ -1,11 +1,26 @@
 const oneYear = 86400 * 365;
+const SENTRY_DSN = process.env.SENTRY_DSN;
 
 module.exports = ({ env }) => {
    const exports = {
       seed: {
          enabled: true,
          resolve: './src/plugins/seed',
       },
+      sentry: {
+         enabled: true,
```

Comment:
```suggestion
         enabled: !!SENTRY_DSN,
```

i.e. if you don't provide the env var the plugin is disabled.

Reply:
Sure, seems correct